src/main/java/games/strategy/triplea/oddsCalculator/ta/OddsCalculator.java

```diff
@@ -5,272 +5,101 @@
 import java.util.Collections;
 import java.util.HashSet;
 import java.util.List;
-import java.util.Map;
-import java.util.Properties;
 import java.util.Set;
-import java.util.concurrent.Callable;
 
-import games.strategy.engine.data.Change;
 import games.strategy.engine.data.CompositeChange;
 import games.strategy.engine.data.GameData;
-import games.strategy.engine.data.PlayerID;
-import games.strategy.engine.data.Territory;
-import games.strategy.engine.data.TerritoryEffect;
 import games.strategy.engine.data.Unit;
-import games.strategy.engine.data.UnitHitsChange;
 import games.strategy.engine.data.UnitType;
 import games.strategy.engine.data.UnitTypeList;
-import games.strategy.engine.data.changefactory.ChangeFactory;
-import games.strategy.engine.delegate.IDelegateBridge;
-import games.strategy.engine.display.IDisplay;
-import games.strategy.engine.framework.GameDataUtils;
-import games.strategy.engine.framework.IGameModifiedChannel;
-import games.strategy.engine.gamePlayer.IRemotePlayer;
-import games.strategy.engine.history.DelegateHistoryWriter;
-import games.strategy.engine.history.IDelegateHistoryWriter;
-import games.strategy.engine.random.IRandomStats.DiceType;
-import games.strategy.engine.random.PlainRandomSource;
-import games.strategy.net.GUID;
-import games.strategy.sound.HeadlessSoundChannel;
-import games.strategy.sound.ISound;
-import games.strategy.triplea.ai.AIUtils;
-import games.strategy.triplea.ai.AbstractAI;
 import games.strategy.triplea.delegate.BattleTracker;
-import games.strategy.triplea.delegate.DiceRoll;
 import games.strategy.triplea.delegate.GameDelegateBridge;
 import games.strategy.triplea.delegate.Matches;
 import games.strategy.triplea.delegate.MustFightBattle;
-import games.strategy.triplea.delegate.dataObjects.CasualtyDetails;
-import games.strategy.triplea.delegate.dataObjects.CasualtyList;
-import games.strategy.triplea.delegate.remote.IAbstractPlaceDelegate;
-import games.strategy.triplea.delegate.remote.IMoveDelegate;
-import games.strategy.triplea.delegate.remote.IPurchaseDelegate;
-import games.strategy.triplea.delegate.remote.ITechDelegate;
-import games.strategy.triplea.ui.display.HeadlessDisplay;
-import games.strategy.triplea.ui.display.ITripleADisplay;
-import games.strategy.util.Match;
+import games.strategy.triplea.oddscalc.DummyDelegateBridge;
+import games.strategy.triplea.oddscalc.OddsCalculatorParameters;
 import games.strategy.util.Tuple;
 
-class OddsCalculator implements IOddsCalculator, Callable<AggregateResults> {
+/**
+ * Simulates battles and reports results of winner/defender win/lose rate.
+ */
+public class OddsCalculator implements IOddsCalculator {
   public static final String OOL_ALL = "*";
-  public static final String OOL_ALL_REGEX = "\\*";
   public static final String OOL_SEPARATOR = ";";
-  public static final String OOL_SEPARATOR_REGEX = ";";
+  private static final String OOL_SEPARATOR_REGEX = ";";
```

Comment:
I don't think 2 constants for the exact same final string are needed

Reply:
I would tend to agree. Though I'm only fixing easy compiler warnings here. Beyond stated scope, I'd rather not expend effort to determine original intent nor complicate the diff further with items that are more than removing unnecessary tracking code revolving around GameData.